back/.rubocop.yml

```diff
@@ -72,6 +72,10 @@ Layout/EndAlignment:
 Layout/TrailingWhitespace:
   AllowInHeredoc: true
 
+Lint/AmbiguousBlockAssociation:
+  AllowedPatterns:
+    - change
+
```

Comment:
Annoying linter errors on a common pattern, decided to allow it instead of fixing.

Lets us do this:
```
expect { service.save_examples([[command, campaign]]) }.to change { EmailCampaigns::Example.all.ids.sort }
```
Instead of having to do this
```
expect { service.save_examples([[command, campaign]]) }.to(change { EmailCampaigns::Example.all.ids.sort })
```